core/src/main/java/org/apache/calcite/sql/fun/SqlFirstLastValueAggFunction.java

```diff
@@ -78,4 +80,16 @@ public RelDataType getReturnType(RelDataTypeFactory typeFactory) {
   @Override public boolean allowsNullTreatment() {
     return true;
   }
+
+  @Override public void unparse(SqlWriter writer, SqlCall call, int leftPrec,
+      int rightPrec) {
+    writer.keyword(getName());
+    final SqlWriter.Frame frame =
+        writer.startList(SqlWriter.FrameTypeEnum.FUN_CALL, "(", ")");
+    call.operand(0).unparse(writer, leftPrec, rightPrec);
+    if (call.operandCount() == 2) {
```

Comment:
nit: Suggest adding a comment describing what we're doing with this if-check since it may not be immediately obvious to a future reader.